src/options/popup.ts

```diff
@@ -31,4 +31,17 @@ const showHotkeys = () => {
   }
 };
 
+const showVersion = () => {
+  const { version } = chrome.runtime.getManifest();
+
+  const versionContainer: HTMLElement | null  = document.querySelector('small[data-version]');
```

Comment:
[minor, nitpick]

why use a `data-attribute` just for selection purpose? :P 
if there isn't any data then we may just use `id` or `className`?

Reply:
Figured we might have this in multiple places, possibly a footer and such. Hence went with the `data-attribute` for this. 

I know we have only one instance now, but would scale for ☝️